oak_functions/abi/src/lib.rs

```diff
@@ -58,7 +57,15 @@ pub struct ReportMetricRequest {
     pub value: i64,
 }
 
-// TODO(#2682): Add TestingMessage to ABI.
+#[derive(Serialize, Deserialize)]
+pub enum TestingRequest {
```

Comment:
I added this TestingExtension and with the TestingRequest/TestingResponse you can now send requests to this testing extension. I hope it is neat if we want our runtime to behave in a certain way---e.g., we could add a testing message for the runtime to sleep for a certain time (from the Wasm module).

It is not really part of UWABI, but I added the testing extension in the context of UWABI because I didn't want to try everything on an existing extension.